dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/model/external/CreateFdcContributionRequest.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.laa.crime.dces.integration.model.external;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
```

Comment:
This comment is about all the classes in `uk.gov.justice.laa.crime.dces.integration.model.external`:

Are these just copied-and-pasted from maat-cd-api? They have very inconsistent lombok annotations (some use `@AllArgsConstructor`, `@NoArgsConstructor`, `@Data`, `@Builder`, etc. - not many use final fields which is one of the reasons to use `@Builder`)

Reply:
Yes, they were originally copy-pasted from maat-cd-api. I have now removed some annotations that I could. Please check the ones in `UpdateConcorContributionStatusRequest` as they might affect your tests.